packages/services/questionnaire/src/app.html

```diff
@@ -1,10 +1,17 @@
 <!doctype html>
-<html lang="en" class="dark">
+<html lang="en" class="dark" style="display: flex">
     <head>
         <meta charset="utf-8" />
         <link rel="icon" href="%sveltekit.assets%/favicon.png" />
         <meta name="viewport" content="width=device-width" />
         %sveltekit.head%
+        <style>
```

Comment:
This the html template which renders all pages. Therefore, if you decide to build upon the questionnaire and add new pages, they will inherit this styling. In order to avoid that, you need to wrap only the page we want to stylize with a div which applies the styling.